arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/halt/StopSection.java

```diff
@@ -47,6 +48,11 @@ public StopSection(Hub hub) {
     // 3 = 1 + max_NON_STACK_ROWS in message call case
     // 5 = 1 + max_NON_STACK_ROWS in deployment case
     super(hub, hub.callStack().currentCallFrame().isMessageCall() ? (short) 3 : (short) 5);
+    final short exceptions = hub.pch().exceptions();
+    checkArgument(
+        Exceptions.none(exceptions),
+        "STOP is incapable of triggering an exception but exceptions = " + exceptions);
+
```

Comment:
What would make sense is to write a `prettyPrint` method for exceptions where we print something

```java
underlying OpCode: <mnemonic>
raw exceptions:
  stackUnderflowException: ✅
  stackOverflowException:
  InvalidOpcodeException: 
  MemoryExpansionException ✅
  OutOfGasException: ✅
  ...
  None ✅
```
(we only print the present exceptions along with `None` (the above example makes no sense, it's just an example.)

Reply:
See https://github.com/Consensys/linea-tracer/issues/1163